angular-spa-ui/js/search-app/search-module/search-ctrl.js

```diff
@@ -131,9 +159,12 @@ module.exports = function (search) {
             params.offset = '0';
```

Comment:
Offset is an integer value. Sometimes operations between Integers and String integers could be unpredictable